sites/tv.vera.com.uy/tv.vera.com.uy.config.js

```diff
@@ -0,0 +1,106 @@
+const axios = require('axios')
+const dayjs = require('dayjs')
+const utc = require('dayjs/plugin/utc')
+const timezone = require('dayjs/plugin/timezone')
+const customParseFormat = require('dayjs/plugin/customParseFormat')
+
+dayjs.extend(utc)
+dayjs.extend(timezone)
+dayjs.extend(customParseFormat)
+
+const API_ENDPOINT = 'https://cds-frontend.vera.com.uy/api-contenidos'
+
+module.exports = {
+  site: 'tv.vera.com.uy',
+  days: 2,
+  async url({ date, channel }) {
+    const session = await loadSessionDetails()
+    if (!session || !session.token) return null
+
+    return `${API_ENDPOINT}/canales/epg/${
+      channel.site_id
+    }?limit=500&dias_siguientes=0&fecha=${date.format('YYYY-MM-DD')}&token=${session.token}`
+  },
+  request: {
+    async headers() {
+      const session = await loadSessionDetails()
+      if (!session || !session.jwt) return null
+
+      return {
+        authorization: `Bearer ${session.jwt}`,
+        'x-frontend-id': 1196,
+        'x-service-id': 3,
+        'x-system-id': 1
+      }
+    }
+  },
+  parser({ content }) {
+    let programs = []
+    let items = parseItems(content)
+    items.forEach(item => {
+      programs.push({
+        title: item.nombre_programa,
```

Comment:
Most are null but there area few programmes that have a sub-title and/or description so worth adding just in case.

```suggestion
        title: item.nombre_programa,
        sub_title: item.subtitle,
        description: item.descripcion_programa,
```